erc20-bridge-token/contracts/BridgeTokenFactory.sol

```diff
@@ -72,19 +67,16 @@ contract BridgeTokenFactory is AccessControlUpgradeable, SelectivePausableUpgrad
 
     // BridgeTokenFactory is linked to the bridge token factory on NEAR side.
     // It also links to the prover that it uses to unlock the tokens.
-    function initialize(address _proofConsumerAddress) external initializer {
+    function initialize(address _proofConsumerAddress, address _tokenImplementationAddress) external initializer {
         require(_proofConsumerAddress != address(0), "Proof consumer shouldn't be zero address");
```

Comment:
The character `'` in the word `shouldn't` broke the github highlighting. Let's remove it to improve the quality of review process.

Reply:
Done